staging/dex-k8s-authenticator/templates/rbac.yaml

```diff
@@ -33,9 +45,10 @@ metadata:
 subjects:
 - apiGroup: ""
   kind: ServiceAccount
-  name: default
+  name: {{ .Release.Name }}
```

Comment:
Should this match the service account name created above? `{{ template "dex-k8s-authenticator.fullname" . }}` https://github.com/mesosphere/charts/pull/1091/files#diff-f78fb2a3e53dd90dcef1e0c8f96e421400e8ad8a956ec28a402a5b3212dae806R6

Reply:
Yes, it most certainly should... <smh>